dozer-sql/src/pipeline/expression/onnx_udf.rs

```diff
@@ -0,0 +1,311 @@
+use crate::pipeline::errors::PipelineError;
+use crate::pipeline::errors::PipelineError::{InvalidType, InvalidValue, OnnxOrtErr, OnnxShapeErr};
+use crate::pipeline::expression::execution::Expression;
+use dozer_types::json_types::JsonValue;
+use dozer_types::log::warn;
+use dozer_types::ordered_float::OrderedFloat;
+use dozer_types::ort::{Session, Value};
+use dozer_types::types::{Field, FieldType, Record, Schema};
+use ndarray::Array;
+use num_traits::FromPrimitive;
+use std::borrow::Borrow;
+use std::ops::Deref;
+
+pub fn evaluate_onnx_udf(
```

Comment:
I'm totally lost with this implementation. For starters:

- Only first input argument is used.
- Onnx model always gets only one tensor as input.
- Input tensor always has only one element.
- Only one output tensor is used.
- Only the first element of the output tensor is used.
- Input tensor type is ignored and always determined by input argument.
- Output tensor type is hardcoded.

Reply:
like we discussed yesterday - will address as stated in the description on this PR